x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiBasicTable,
+  EuiText,
+  EuiTableFieldDataColumnType,
+  EuiToolTip,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  FormattedMessage,
+  FormattedDate,
+  FormattedTime,
+  FormattedNumber,
+  FormattedRelative,
+} from '@kbn/i18n/react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+import { usePageId } from '../use_page_id';
+import {
+  selectIsLoading,
+  selectPageIndex,
+  selectPageSize,
+  selectPolicyItems,
+  selectTotal,
+} from '../../store/policy_list/selectors';
+import { usePolicyListSelector } from './policy_hooks';
+import { PolicyListAction } from '../../store/policy_list';
+import { PolicyData } from '../../types';
+import { TruncateText } from '../../components/truncate_text';
+
+interface TableChangeCallbackArguments {
+  page: { index: number; size: number };
+}
+
+const TruncateTooltipText = styled(TruncateText)`
+  .euiToolTipAnchor {
+    display: block;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+`;
+
+const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
+  // If date is greater than or equal to 24h (ago), then show it as a date
+  // else, show it as relative to "now"
+  return Date.now() - date.getTime() >= 8.64e7 ? (
+    <>
+      <FormattedDate value={date} year="numeric" month="short" day="2-digit" />
+      {' @'}
+      <FormattedTime value={date} />
+    </>
+  ) : (
+    <>
+      <FormattedRelative value={date} />
+    </>
+  );
+};
+
+const renderDate = (d: string, _item: PolicyData) => (
```

Comment:
this is a bit of a nit but one char variable names are pretty meh

Reply:
Agreed 😄  . I will update it. Thanks.